cmake/utils.cmake

```diff
@@ -99,6 +99,7 @@ macro(shaderc_get_transitive_libs target out_list)
     get_target_property(libtype ${target} TYPE)
     # If this target is a static library, get anything it depends on.
     if ("${libtype}" STREQUAL "STATIC_LIBRARY")
+      list(INSERT ${out_list} 0 "${target}")
```

Comment:
Should the blurb mention this?  Or is it obvious that `target` is its own dependency?